

const adminLogin=(req,res)=>{
    try{}
    catch(err){
        return res.status(500).json({message:"server returned an error"})
    }
}
const getAllTasks=(req,res)=>{
    try{}
    catch(err){
        return res.status(500).json({message:"server returned an error"})
    }
}
const updateTask=(req,res)=>{
    try{}
    catch(err){
        return res.status(500).json({message:"server returned an error"})
    }
}
const deleteTask=(req,res)=>{
    try{}
    catch(err){
        return res.status(500).json({message:"server returned an error"})
    }
}
const addTask=(req,res)=>{
    try{}
    catch(err){
        return res.status(500).json({message:"server returned an error"})
    }
}

export {adminLogin,getAllTasks,updateTask,deleteTask,addTask}